.gitignore

```diff
@@ -5,3 +5,5 @@
 *.o
 dustmite
 /cov/
+.dub
+libdustmite.a
```

Comment:
I don't understand why this is needed. If it's a temporary file, it should be created in `.dub`.

Reply:
Ahh, correct. Removed that line.